### PR TITLE
GSYE-637: Added gsy_energy_bill_revenue, gsy_energy_bill_excl_fees an…

### PR DESCRIPTION
**Pull request by spyrostz**
…d gsy_total_benefit calculation.

## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/models/area/scm_manager.py [hunk 1]
```diff
@@ -310,6 +313,22 @@ def gsy_energy_bill_excl_revenue_without_fees(self):
         return (self.gsy_energy_bill_excl_revenue - self.grid_fees - self.tax_surcharges
                 - self.fixed_fee - self.marketplace_fee - self.assistance_fee)
 
+    @property
+    def gsy_energy_bill_revenue(self):
+        """Energy bill of the home excluding revenue."""
```
Comment (hannesdiedrich) [inline src/gsy_e/models/area/scm_manager.py:318]: The docstring does not comply to the implementation. 
How about:
```suggestion
        """Total revenue from gird and community."""
```

File: src/gsy_e/models/area/scm_manager.py [hunk 2]
```diff
@@ -310,6 +313,22 @@ def gsy_energy_bill_excl_revenue_without_fees(self):
...
+        return self.earned_from_grid + self.earned_from_community
+
+    @property
+    def gsy_energy_bill_excl_fees(self):
+        """Energy bill of the home excluding revenue."""
```
Comment (hannesdiedrich) [inline src/gsy_e/models/area/scm_manager.py:323]: Here too, I think the following should apply, right?
```suggestion
    def gsy_energy_bill_excl_fees(self):
        """Energy bill of the home excluding fees."""
```

File: src/gsy_e/models/area/scm_manager.py [hunk 3]
```diff
@@ -310,6 +313,22 @@ def gsy_energy_bill_excl_revenue_without_fees(self):
...
+        return (self.gsy_energy_bill - self.grid_fees - self.tax_surcharges
```
Comment (hannesdiedrich) [inline src/gsy_e/models/area/scm_manager.py:324]: Can you please remind me of the sign convention of `gsy_energy_bill` ? What if this is negative? Does this mean the user has a credit balance? If yes, the fees should be added here, right?

**Review by hannesdiedrich — COMMENTED** [review]
Some questions and comments, LGTM once addressed

File: src/gsy_e/models/area/scm_manager.py (hunk 1 above)
Comment (spyrostz) [reply src/gsy_e/models/area/scm_manager.py:318]: Good catch, thanks !

File: src/gsy_e/models/area/scm_manager.py (hunk 2 above)
Comment (spyrostz) [reply src/gsy_e/models/area/scm_manager.py:323]: Indeed, thanks!

File: src/gsy_e/models/area/scm_manager.py (hunk 3 above)
Comment (spyrostz) [reply src/gsy_e/models/area/scm_manager.py:324]: The energy bill includes the fees right from the start (essentially its default value is not 0 but `monthly_fee + assistance_fee + fixed_fee`) therefore the fees should be subtracted from it.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**